packages/common/package.json

```diff
@@ -27,6 +27,7 @@
     "url": "https://github.com/AudiusProject/audius-client/issues"
   },
   "dependencies": {
+    "@audius/sdk": "*",
```

Comment:
okay so turbo actually has us specify deps? personally like this much better.

Reply:
Yeah exactly! It wasn't necessary for npm workspaces, but this is how turbo builds the dependency graph